core/src/main/scala/akka/persistence/cassandra/journal/TagWriter.scala

```diff
@@ -139,7 +139,7 @@ import scala.concurrent.duration._
       // reached to force a flush or that the batch size is met
       val (newTagPidSequenceNrs, events) =
         assignTagPidSequenceNumbers(payload.toVector, tagPidSequenceNrs)
-      log.debug("Assigned tag pid sequence nrs: {}", events)
+      log.debug("Assigned tag pid sequence nrs: {}", newTagPidSequenceNrs)
```

Comment:
events contains serialized blob stuff, I think the Map of TagPidSequenceNrs is more interesting